storybookjs/storybook
# Remove vite-plugin-svelte-kit when detected

Issue: #19280

## What I did

- Remove `vite-plugin-svelte-kit` when detected as per discussions in #19280
- Re-enable SvelteKit repros

## How to test

- Create the `svelte-kit/skeleton-js` sandbox
- Ensure that it both runs in dev and it builds
